pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -333,6 +333,15 @@ in
     dependencies = [ self.blink-cmp ];
   };
 
+  blink-cmp-words = super.blink-cmp-words.overrideAttrs {
+    dependencies = [ self.blink-cmp ];
+    nvimSkipModules = [
+      "blink-cmp-words.dictionary"
+      "blink-cmp-words.source"
+      "blink-cmp-words.thesaurus"
```

Comment:
Can you add a short comment above these to explain why they need to be skipped please?